docs/intro.md

```diff
@@ -1,3 +1,7 @@
+```{eval-rst}
+.. include:: ../README.rst
+```
+
```

Comment:
I think it's fantastic to have a good README on GitHub. And the better the README, the more reason to repeat it as a documentation start page. To me, the README is usually a perfect fit as the start page in the documentation -- not for all projects, but for the small/medium-sized documentation projects.

Larger projects probably want a start page that contains deep links on the documentation start page.

In for "DRY", I think the include pattern is useful to spread. Under current circumstances, I would consider it "best practice".

It's also an organic way to grow documentation: You start off with a README, and as various headlines grow into separate sections, you move them out. _Always complete, never finished._ There's a good description here: https://diataxis.fr/how-to-use-diataxis/#allow-your-work-to-develop-organically

What alternative would you like to see?

Reply:
I'm not saying that having a README is bad. I'm saying that including files from _outside the `docs/`_ directory is not good practice to recommend, IMO, and could produce unexpected issues that are hard to debug.

Reply:
I've added more guidance on this btw.. see the current README.rst.

I would think it's an opportunity missed if the example project has a README that is either 1) not in the docs or 2) duplicated in the docs.

Reply:
I would also note it makes a weird UX in the rendered documentation pages. You land on the front page of the example project, and all the links take you outside of the documentation into the actual GH repo -- which seems a bit odd and not what I'd expect.

Reply:
@ericholscher @benjaoming 

> > You land on the [front page](https://example-jupyter-book.readthedocs.io/en/bootstrap/intro.html#read-the-docs-tutorial) of the example project, and all the links take you outside of the documentation into the GH repo
>
> I kind of liked that it took the reader to the source code of said files. For instance, the _config.yml and requirements.txt aren't part of the rendered documentation.

Eric has a point here to me. I didn't think too much about this previously but now he mentions it, it sounds weird to me as well. We can probably solve this problem by using `.. literalinclude::` directive to include these files as part of the documentation itself. So, when you click on those files you can see them rendered inside the documentation with the same look and feel. Example: https://test-builds.readthedocs.io/en/git-lfs/conf.html

Reply:
Yea, I agree that if we want to showcase these in the rendered docs, it should just inline them. But I think we're getting overly cute to start. It seems like the rendering can just be an example of the rendered docs, and not try to explain the explain too deeply, just pointing to GH to do that job?

Reply:
I understand that this could be our separate scopes (overlaps forgiven) to avoid piling up too much work for now:

* GitHub: Explains entrypoints in the source code and how to use the Example Project
* Rendered example project: Showcases content and features that are particular to the example

Reply:
Will resolve this thread, since we did manage to remove the `include::` directive already :+1: Thanks for inputs :100: